app/views/projects/versions/view.scala.html

```diff
@@ -115,6 +125,16 @@ <h1 class="pull-left">@v.v.versionString</h1>
                                 </ul>
                             </div>
 
+                            @if(request.hasUser && request.data.globalPerm(ViewLogs)) {
+                                <button class="btn btn-alert dropdown-toggle" type="button" id="admin-version-actions" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
```

Comment:
wrap the `button` and `ul dropdown-menu` around `<div class="dropdown dropdown-menu-right" style="display: inline-block">
` to fix the alignment issue.